packages/monaco/src/browser/monaco-workspace.ts

```diff
@@ -147,12 +147,12 @@ export class MonacoWorkspace extends BaseMonacoWorkspace implements lang.Workspa
             promises.push(this.monacoModelResolver.createModelReference(edit.resource).then(reference => {
                 const model = reference.object.textEditorModel;
                 const range = edit.range;
-                model.applyEdits([{
+                model.pushEditOperations([], [{
```

Comment:
https://github.com/theia-ide/theia/blob/master/packages/monaco/src/browser/monaco-editor.ts#L48 and then you can filter by the model `editor.getControl().model`